packages/web-components/src/components/rux-indeterminate-progress/rux-indeterminate-progress.tsx

```diff
@@ -0,0 +1,80 @@
+import { Component, Host, h, Element } from '@stencil/core'
+
+/**
+ * @part container - The outermost div encasing rux-indeterminate-progress which is responsible for the overall size and the outermost border.
+ * @part animated-spinner - The second div in the heirarchy which is responsible for the spinning conic-gradient and the gap between the spinner and the containers's border.
+ * @part inner-spinner-gap - The third div in the heirarchy which controls the gap between the animated-spinner and the inner-circle's border.
+ * @part inner-circle - The fourth div in the heirarchy which controls the inner-most circle.
+ */
+
+@Component({
+    tag: 'rux-indeterminate-progress',
+    styleUrl: 'rux-indeterminate-progress.scss',
+    shadow: true,
+})
+export class RuxIndeterminateProgress {
+    @Element() el!: HTMLRuxIndeterminateProgressElement
+
+    private small: boolean = false
+
+    private _setSmall() {
```

Comment:
@markacianfrani what do you think of the setSmall and determineMeasurement funcs? Worth my time? Any other unit measurements I should account for?

Reply:
![Screen Shot 2022-05-06 at 9 58 20 AM](https://user-images.githubusercontent.com/1091342/167147943-51066a91-bc6d-466b-b1fc-2b5cee92f3cb.png)
![Screen Shot 2022-05-06 at 9 58 13 AM](https://user-images.githubusercontent.com/1091342/167147946-7c52bd95-0a4f-4b08-bbf5-c5a88bbd1f6a.png)

is this the only noticeable difference between the two at 20px? I'd say just leave it out entirely for now until we have a better design story